src/nni_manager/training_service/pai/paiK8S/paiK8STrainingService.ts

```diff
@@ -144,66 +143,60 @@ class PAIK8STrainingService extends PAITrainingService {
             throw new Error('trial config is not initialized');
         }
         const jobName = `nni_exp_${this.experimentId}_trial_${trialJobId}`
```

Comment:
I think showing `experimentName` in jobName would be more user-friendly.

Reply:
Experiment Id will be shown in NNICTL commands, it is easier to get for users..